.gitignore

```diff
@@ -20,3 +20,7 @@ bin/
 
 #file generated at build time by message compiler
 agentmsg.h
+
+# go-generated files
+datadog.yaml
+dogstatsd.yaml
```

Comment:
You could use that "insert final new line" setting in your editor 😉

Reply:
`"files.insertFinalNewline": true,` VS code doesn't cease to amaze me 😄